ygot/diff.go

```diff
@@ -426,6 +437,24 @@ func hasIgnoreAdditions(opts []DiffOpt) *IgnoreAdditions {
 	return nil
 }
 
+// The RespectPresenceContainers DiffOpt indicates that presence containers
+// should be respected in the diff output.
+// This option was added to ensure we do not break backward compatibility.
+type WithRespectPresenceContainers struct{}
+
+func (*WithRespectPresenceContainers) IsDiffOpt() {}
+
+// hasIgnoreAdditions returns the first IgnoreAdditions from an opts slice, or
+// nil if there isn't one.
```

Comment:
During a self review on my phone right now, I can see this is missing a proper docstring. I’ll fix Monday morning!